Topics/Tech_Stacks/Flutter.md

```diff
@@ -0,0 +1,82 @@
+2. **Large and Weighty Apps**
+   A loophole that can’t be dismissed is the size of the large size of the applications under development. Software developers working with this toolkit may find it difficult to work with large files. This can cause them to choose a lighter alternative.
+3. **Limited number of third-party libraries**
+   Flutter being relatively new cannot be compared to native programming languages. Developers still need to spend more time building as many libraries as possible.
+
+   Source [Waverley Software](https://waverleysoftware.com/blog/why-use-flutter-pros-and-cons/#:~:text=Flutter%20allows%20developers%20to%20build,application%20development%20are%20much%20lower.)
+
+## Creating your first flutter app
+
+The following is a summary from the offical [Flutter Docs](https://docs.flutter.dev/get-started/).
+
+1. **Install Flutter**
+
+   1. [Windows](https://docs.flutter.dev/get-started/install/windows)
+   2. [macOS](https://docs.flutter.dev/get-started/install/macos)
+   3. [Linux](https://docs.flutter.dev/get-started/install/linux)
+   4. [ChromeOS](https://docs.flutter.dev/get-started/install/chromeos)
+
+2. **Install an editor**
+   While you can use any editor. These instructions are for setting up Flutter on VS Code
+
+   1. Install VS Code for your respective platform. [Installation Link](https://code.visualstudio.com/download).
+   2. Open a browser and go to the [Flutter extension page](https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter) on the Visual Studio Marketplace.
+   3. Click Install. Installing the Flutter extension also installs the Dart extension.
+
+3. **Create the app**
+
+   1. Invoke **View > Command Palette**. This can be done with `ctrl shift p` on Windows or `cmd shift p` on Mac.
+      <img width="636" alt="VS Code Command Palette" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/65613404/9bdf3ef1-ad9b-4848-b6bb-723f85462c36">
+   2. Type "flutter", and select the **Flutter: New Project**
+      <img width="644" alt="Flutter new project" src="https://github.com/learning-software-engineering/learning-software-engineering.github.io/assets/65613404/47cf0a8d-25a5-4076-ad6d-54cc8197384d">
+   3. Select **application**
+   4. Create or select the parent directory for the new project folder.
+   5. Enter a project name, such as `my_app`, and press **Enter**.
+   6. Wait for project creation to complete and the `main.dart` file to appear.
+      The above commands create a Flutter project directory called my_app that contains a simple demo app that uses [Material Components](https://m3.material.io/components).
+   7. Your file structures should look as follows.
```

Comment:
The image is being rendered on the same line as the text. Can you add a newline so it looks better?